docs/api-reference.md

```diff
@@ -3,25 +3,623 @@ id: api-reference
+| store         | A store object.                    |
+| readModelName | A Read Model whose data to remove. |
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/custom-readmodel-sample/connector.js#drop)
+```js
+const drop = async (store, readModelName) => {
+  safeUnlinkSync(`${prefix}${readModelName}.lock`)
+  safeUnlinkSync(`${prefix}${readModelName}`)
+}
+```
+
+<!-- prettier-ignore-end -->
+
+### dispose
+
+Forcefully disposes all unmanaged resources provided by this connector.
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/custom-readmodel-sample/connector.js#dispose)
+```js
+const dispose = async () => {
+  for (const readModelName of readModels) {
+    safeUnlinkSync(`${prefix}${readModelName}.lock`)
+  }
+  readModels.clear()
+}
+```
+
+<!-- prettier-ignore-end -->
+
+## Read Model Store Interface
+
+The table below lists functions that you can use to communicate with a Read Model store through a `store` object.
+
+| Function Name               | Description                                       |
+| --------------------------- | ------------------------------------------------- |
+| [defineTable](#definetable) | Defines a new table within the store.             |
+| [find](#find)               | Searches for data items.                          |
+| [findOne](#findone)         | Searches for a single data item.                  |
+| [count](#count)             | Returns the count of items that meet a condition. |
+| [insert](#insert)           | Inserts an item into a table.                     |
+| [update](#update)           | Searches for data items and updates them.         |
+| [delete](#delete)           | Deletes data items.                               |
+
+### defineTable
+
+Defines a new table within the store.
+
+#### Arguments
+
+| Argument Name    | Description                                         |
+| ---------------- | --------------------------------------------------- |
+| tableName        | The new table's name.                               |
+| tableDescription | An object that describes the new table's structure. |
+
+#### Example
+
+```js
+Init: async store => {
+  await store.defineTable('Stories', {
+    indexes: { id: 'string', type: 'string' },
+    fields: [
+      'title',
+      'text',
+      'link',
+      'commentCount',
+      'votes',
+      'createdAt',
+      'createdBy',
+      'createdByName'
+    ]
+  })
+```
+
+### find
+
+Searches for data items that based on the specified expression.
+
+#### Arguments
+
+| Argument Name    | Description                                 |
+| ---------------- | ------------------------------------------- |
+| tableName        | A table name.                               |
+| searchExpression | An object that defines a search expression. |
+| fieldList        | A list of fields to fetch.                  |
+| sort             | The sort order.                             |
```

Comment:
Why use the definite article here, but not for the other descriptions?

Reply:
changed to 'a sort order'